app/lib/dags/host_filter.json.jbuilder

```diff
@@ -2,6 +2,10 @@ json.name attr[:dag_name]
 
 json.output_dir_s3 "s3://#{attr[:bucket]}/samples/#{attr[:project_id]}/#{attr[:sample_id]}/results"
 
+compute_insert_size_metrics = attr[:fastq2] and
+  attr[:host_genome] == 'human' and
+  attr[:sample_template] == 'DNA'
```

Comment:
All `sample_*` columns are deprecated. You have to go via Metadatum / MetadataField